README.md

```diff
@@ -114,7 +114,7 @@ The MapTable `viz` declaration in the above example is a chain of functions. The
 - [viz.map(mapOptions)](#map) with `mapOptions` as a JS dictionary. You can add/remove it of you want a map on your visualization.
 - [viz.filters(filtersOptions)](#filters) with `filtersOptions` as a JS dictionary. You can add/remove it of you want filters on your visualization.
 - [viz.table(tableOptions)](#table) with `tableOptions` as a JS dictionary. You can add/remove it of you want a table on your visualization.
-- [viz.render()](#render) that closes the chain and renders the visualization. Don't forget this!
+- [viz.render([onComplete])](#render) that closes the chain and renders the visualization. Don't forget this! It can take an optional callback function `onComplete`, that's executed when MapTable finishes rendering its components.
```

Comment:
maybe make this:
> viz.render([onComplete]) that closes the chain and renders the visualization. Don't forget this! It can take an optional callback function onComplete, that's executed when MapTable finishes rendering its components. For exmple if you have `function alertTest(){ alert('test!'); }` you would call it with `viz.render(alertTest)`.